internal/commands/app/initialize/errors.go

```diff
@@ -0,0 +1,7 @@
+package initialize
+
+type errProjectExists struct{}
+
+func (err errProjectExists) Error() string { return "a project already exists" }
```

Comment:
will we need any additional contextual information for this error?

Reply:
i hope not? though that's likely answered in practice.  it _should_ be showing up as a fatal error that terminates command execution and says "ERROR app init failed: a project already exists"

we have some work currently scheduled that allow for errors to associate followup commands and/or documentation links that this would probably benefit from ("Try running the same command inside an empty directory" or something along those lines)

Reply:
depending on how we refactor PrintUsage, this might not be needed as its own error struct.

Reply:
Moved `PrintUsage` to `DisableUsage` in the refactored approach